lighthouse/src/main.rs

```diff
@@ -75,6 +76,12 @@ fn main() {
         std::env::set_var("RUST_BACKTRACE", "1");
     }
 
+    let env_filter = EnvFilter::builder()
+        .with_default_directive(LevelFilter::INFO.into())
+        .from_env_lossy();
+
+    tracing_subscriber::fmt().with_env_filter(env_filter).init();
+
```

Comment:
we'll want to move the tracing subscriber further downstream. see `common/logging/lib.rs` for some examples. if you have any questions let me know